src/test/resources/testsData.properties

```diff
@@ -1,3 +1,15 @@
 userName=konstantinliauchuk
```

Comment:
Name should be divided by dot. Example user.name= konstantinliauchuk

Reply:
fixed